src/OpenFOAM/meshes/meshShapes/faceData/faceData.H

```diff
@@ -118,6 +118,9 @@ public:
              return start_;
         }
 
+        //- Return the points corresponding to this face
+        inline void points(const point* meshPoints, point* facePoints) const;
```

Comment:
As I commented earlier this method is not necessary since the center of the face should be calculated manually.